### DISCUSSION TITLE
We need a better name for the project

### DISCUSSION THREAD

**Discussion by toniengelhardt** [toniengelhardt/merklin#3] [General]
How should we name the project?

**Comment by RyanPaulGannon** [comment]
Do you think it should have some kind of reference to the Vue/Nuxt aspect of the project? I heard someone use 'Nuxtify' as a verb on a YouTube video, which I thought it was cool (not an idea for this, however 😃). Or are we just going for something more web3-esque ...

**Reply by toniengelhardt** [reply]
Not such a big fan of vueifying all the names tbh, would prefer smth snappy and web3y.

**Reply by RyanPaulGannon** [reply]
How's about the one you posted on Twitter?

**Reply by toniengelhardt** [reply]
You mean Merklin? Not sure about that one lol

**Reply by RyanPaulGannon** [reply]
Hahaha fair 😅

This may be construed as 'Vueify-ing' so not offended if it's not liked, but the French for money, Argent?

**Reply by toniengelhardt** [reply]
There is already a wallet with that name.

**Comment by toniengelhardt** [comment]
@RyanPaulGannon for now I just renamed it to Merklin. We can still rename it again later, but I thought it would be cool to have it up on a domain and [merklin.xyz](https://merklin.xyz) was available and cheap. Also, I like the magic theme and I have already a sweet idea for a logo, like a Merlin wizard hat with a merkle tree pattern on it.

**Comment by RyanPaulGannon** [comment]
I quite like the name, and I'm intrigued to see this logo!

**Reply by toniengelhardt** [reply]
It's nice and short. I tried StableDiffusion to generate one, but only crap came out...

**Reply by RyanPaulGannon** [reply]
Haha, what did you use to register a domain with .xyz?

**Reply by toniengelhardt** [reply]
Always registering domains with [gandi.net](http://gandi.net).

**Reply by RyanPaulGannon** [reply]
Cool, will take a look.
